independent-projects/bootstrap/pom.xml

```diff
@@ -71,7 +71,7 @@
         <jboss-logmanager-embedded.version>1.0.11</jboss-logmanager-embedded.version>
         <slf4j-jboss-logmanager.version>1.1.0.Final</slf4j-jboss-logmanager.version>
         <slf4j-api.version>1.7.36</slf4j-api.version>
-        <graal-sdk.version>22.3.2</graal-sdk.version>
+        <graal-sdk.version>23.0.0</graal-sdk.version>
```

Comment:
this makes it only build on java 11.
afaics it only brings in the annotations - that library shouldn't be java 17 bound IMO.

I've pinged on graalvm slack what their recommendation is.

In worst case we stay on the old version for this as it just brings substitution annotations.

Reply:
one piece of evidence to stay on 22.3.2 is that there are no API differences:
https://diff.revapi.org/?groupId=org.graalvm.sdk&artifactId=graal-sdk&old=22.3.2&new=23.0.0